test/chargebeex/subscription_test.exs

```diff
@@ -8,6 +8,257 @@ defmodule Chargebeex.SubscriptionTest do
 
   setup :verify_on_exit!
 
+  describe "retrieve" do
+    test "with bad authentication should fail" do
+      unauthorized = Common.unauthorized()
+
+      expect(
+        Chargebeex.HTTPClientMock,
+        :get,
+        fn url, body, headers ->
+          assert url == "https://test-namespace.chargebee.com/api/v2/subscriptions/1234"
+          assert headers == [{"Authorization", "Basic dGVzdF9jaGFyZ2VlYmVlX2FwaV9rZXk6"}]
```

Comment:
Move to module variable `@headers` ?

Reply:
We have `[{"Authorization", "Basic dGVzdF9jaGFyZ2VlYmVlX2FwaV9rZXk6"}]` headers for `GET` requests and 
`[{"Authorization", "Basic dGVzdF9jaGFyZ2VlYmVlX2FwaV9rZXk6"}, {"Content-Type", "application/x-www-form-urlencoded"}]` for `POST` requests. Should we create a `@get_headers` and a `@post_headers` module attributes? If so IMO we should refactor the other test modules in order to have the same pattern, WDYT?

Reply:
And yes ! :smile: